src/home-page/home-page.component.scss

```diff
@@ -0,0 +1,64 @@
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("https://www.cray.com/blog/wp-content/uploads/2015/09/Weather-Blog-Image.jpg");
```

Comment:
I guess it's for dev purposes, but avoid fetching files from other servers. Instead download it, add to assets folder and fetch it from there.